### ISSUE TITLE
Mouse pointer is very slow when running Jubula tests. 

### ISSUE THREAD

**Issue by Montepaso** [adoptium/adoptium-support#443]
## Summary

Mouse pointer is very slow when running Jubula tests. 

It started after Temurin-11.0.12+7 update, with version 11.0.11+9 and lower it works fine. Anything that is 11.0.12+7 and up is failing. 

## Steps to reproduce

I cannot reproduce it with the sample application, with sample Jubula app it works fine. But the app i am trying to test is  failing, although I know that no code changes were made, just Java updated. 

i start app with this command: 
"C:\PEC\java64\jre\bin\javaw.exe" -DNODB2IGNORE -DDB2INSTANCE=DB2 -DDB2V9MIN=9.0.0.0 -DDB2OSCMIN=1.1.4.0 -DDB2OEMIN=1.2.1.0 -DDB2PEJVMOPTIONS="-Djava.util.prefs.syncInterval=2000000 -Dsun.java2d.noddraw=true -Djava.util.Arrays.useLegacyMergeSort=true" -DPECLASSPATH=db2.jar;common.jar;fpesrv.jar;ITLMToolkit.jar;db2jcc4.jar;db2jcc_license_cisuz.jar;jlog.jar;images.jar;com.ibm.datatools.perf.repository.api.basicprofile_531.0.8523.jar;com.ibm.datatools.perf.repository.api.config_531.0.8523.jar;com.ibm.datatools.perf.repository.api.access_531.0.8523.jar;ext/ibmjsse.jar;ext/jhall.jar;ext/jhtools.jar;ext/mail.jar;ext/smtp.jar;ext/IBMJCEfw.jar;ext/activation.jar;ext/imap.jar;ext/jh.jar;ext/jhbasic.jar;ext/jsearch.jar;ext/local_policy.jar;ext/mailapi.jar;ext/pop3.jar;ext/US_export_policy.jar -DPEMAINCLASS=com.ibm.db2pm.sysovw.main.SysOvwLauncher -jar launcher.jar

I guess something has changed related to the mouse control but i cant find anything. I have wasted a week on this without results. any thought would be very usefull!

**Comment by karianna** [comment]
I'm not sure what Jubulu tests are - can you give us some more context there?

**Comment by Montepaso** [comment]
Jubula is automated test tool we use to test java apps. It controls the mouse and reads app field values.  https://www.bredex.de/jubula#jubulaforwindows

**Comment by karianna** [comment]
Have you reported this issue to Jubula?  I'd be interested to hear from
them if this is a common issue.

Cheers,
Martijn

**Comment by Montepaso** [comment]
Yes, i did, but no one responded yet. No changes from Jubula side.

**Comment by sxa** [comment]
Link to issue: https://www.eclipse.org/forums/index.php/t/1109633/

**Comment by karianna** [comment]
Still an open issue on the Jubula boards.  I looked at the release notes for 11.0.12 (https://foojay.io/java-11/?tab=allissues&quarter=072021&version=11.0.12) but couldn't spot anything obvious except for one fix to drag and drop on windoes.
